app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    created_people = []
+    for person in people:
+        new_person = Person(person["name"], person["age"])
+        created_people.append(new_person)
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
```

Comment:
You could shorten the line and avoid line breaks by allocating a `person_instance` variable and using get with walrus operator. Apply the same change to the `husband` condition as well
```suggestion
    for person in people:
        person_instance = Person.people[person["name"]]
        if wife_name := person.get("wife"):
           person_instance.wife = Person.people[wife_name]
```